UnitTests/ConditionalOperatorsHelperTests.cs

```diff
@@ -0,0 +1,145 @@
+﻿using System;
+using NUnit.Framework;
+using static ClassLibrary.ConditionalOperatorsHelper;
+
+namespace UnitTests
+{
+    class ConditionalOperatorsHelperTests
+    {
+        [Test]
+        public static void MathOperationAfterComparingNumbers_WhenAMoreThanB_ShouldCalculateSum()
+        {
+            double result = MathOperationAfterComparingNumbers(5, 6.5);
+
+            Assert.AreEqual(-1.5, result);
+        }
+
+        [Test]
+        public static void MathOperationAfterComparingNumbers_WhenALessThanB_ShouldCalculateMinus()
```

Comment:
Two equivalent tests. This and line 10.
Also look at 25 line

Reply:
fixed